doc/dependency_decisions.yml

```diff
@@ -425,68 +425,68 @@
     :when: 2018-01-17 12:06:07.468492000 Z
 - - :whitelist
   - MIT/X11
-  - :who:
-    :why:
+  - :who: 
```

Comment:
Can we remove the extra spaces?

Reply:
I believe this is auto-generated by license_finder :S

Reply:
It is

Reply:
unacceptable